tratto/src/test/java/star/tratto/oraclegrammar/custom/ParserTest.java

```diff
@@ -68,8 +68,10 @@ public void detectPersistentErrorColumnsTest() {
                     .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)));
 
             if (!columnsToCheck.isEmpty()) {
-                fail("The following oracle contains a sequence of tokens such that the same column is marked with an error five or more times:\n"
-                        + stringOracle + "\n\n" + "Columns marked with errors three or more times:\n" + columnsToCheck);
+                fail(String.join(System.lineSeparator("The following oracle contains a sequence of tokens such that the same column is marked with an error five or more times:",
+                                                      stringOracle,
+                                                      "Columns marked with errors three or more times:",
+                                                      columnsToCheck)));
```

Comment:
The project fails to build both in my computer and CI due to this.

See https://github.com/AML14/tratto/runs/15070008431

```
ParserTest.java:[71,40] method lineSeparator in class java.lang.System cannot be applied to given types;
```